pennylane/optimize/gradient_descent.py

```diff
@@ -106,8 +106,8 @@ def step(self, objective_fn, *args, grad_fn=None, **kwargs):
 
     @staticmethod
     def compute_grad(objective_fn, args, kwargs, grad_fn=None):
-        r"""Compute gradient of the objective_fn at the point x and return it along with the
```

Comment:
The docstring conventions discourage the use of variable names in the first line/summary line of the docstring. Hence replacing `objective_fn` with 'objective function'.